client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -59,9 +59,9 @@ const ConnectedSingleSiteJetpack = connectOptions( ( props ) => {
 			{ ! requestingSitePlans && currentPlan && ! isPartnerPlan && (
 				<UpsellNudge
 					forceDisplay
-					title={ translate( 'Get unlimited premium themes' ) }
+					title={ translate( 'Upload your own themes' ) }
 					description={ translate(
-						'In addition to our collection of premium themes, get comprehensive WordPress' +
+						'In addition to uploading your own themes, get comprehensive WordPress' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'In addition to our collection of premium themes, get comprehensive WordPress security, real-time backups, and unlimited video hosting.' )` *ES Score: 7*